### PR TITLE
Feature/wsdroid 548 display songbook below song

### PR DESCRIPTION
**Pull request by sskjames**

### CODE REVIEW

**Review by sibiarunachalam — APPROVED** [review]

File: app/src/main/java/org/worshipsongs/domain/DragDrop.kt [hunk 1]
```diff
@@ -16,25 +16,24 @@ import java.util.ArrayList
 
 open class DragDrop
 {
-    var id: Long = 0
+    var sortOrder: Long = 0
```
Comment (VigneshPalanisamy) [inline app/src/main/java/org/worshipsongs/domain/DragDrop.kt:19]: sortOrder is looks like Ascending or descending order. If can change the variable name @sskjames

File: app/src/main/java/org/worshipsongs/domain/DragDrop.kt [hunk 2]
```diff
@@ -16,25 +16,24 @@ import java.util.ArrayList
...
 
     constructor()
     {
-
     }
 
     constructor(id: Long, title: String, checked: Boolean)
     {
-        this.id = id
+        this.sortOrder = id
```
Comment (VigneshPalanisamy) [inline app/src/main/java/org/worshipsongs/domain/DragDrop.kt:29]: please change the constructor id as sortOrder

File: app/src/main/java/org/worshipsongs/domain/DragDrop.kt (hunk 1 above)
Comment (sskjames) [reply app/src/main/java/org/worshipsongs/domain/DragDrop.kt:19]: I was confused with the previous name and thought it to be a song's id and went in a wrong direction Vicky.

Comment (VigneshPalanisamy) [reply app/src/main/java/org/worshipsongs/domain/DragDrop.kt:19]: yes James it confusing only changing it is good only...

**Review by VigneshPalanisamy — APPROVED** [review]

**Pull request merged**